### PR TITLE
Add GasPrice to BackupCodes

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

**Comment by cehali** [comment]
Before:
image
After:
image

**Comment by cehali** [comment]
Before:
image
After:
image

**Comment by cehali** [comment]
Before:
image
After:
image

**Comment by cehali** [comment]
Before:
image
After:
image

**Comment by cehali** [comment]
Mobile:
image
image

File: universal-login-sdk/lib/api/DeployedWallet.ts [hunk 1]
```diff
@@ -95,7 +92,7 @@ export class DeployedWallet implements ApplicationWallet {
     return this.sdk.getGasModes();
   }
 
-  async generateBackupCodes(): Promise<BackupCodesWithExecution> {
+  async generateBackupCodes(transactionDetails: Partial<Message>): Promise<BackupCodesWithExecution> {
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/api/DeployedWallet.ts]: `Partial<Message>` -> `PaymentOptions`

Comment (cehali) [reply universal-login-sdk/lib/api/DeployedWallet.ts]: Refactoring in: https://github.com/UniversalLogin/UniversalLoginSDK/pull/1516

**Comment by cehali** [comment]
New version (I got rid of text 'Generate new codes and keep them safe'. I think it isn't much informative and the button is self-explanatory):
image
image
image
image

Mobile:
image
image

**Pull request merged**